lib/linux_admin/disk.rb

```diff
@@ -69,8 +69,8 @@ def size
         size = nil
         out = run!(cmd(:fdisk), :params => {"-l" => nil}).output
         out.each_line { |l|
-          if l =~ /Disk #{path}: ([0-9\.]*) ([KMG])B.*/
-            size = str_to_bytes($1, $2)
+          if l =~ /Disk #{path}: .*B, (\d+) bytes/
+            size = $1.to_f
```

Comment:
Why `.to_f`?  Can you have fractional bytes?  Or is this just to keep the code consistent with returning a float beforehand?